server/routes/checkAnswersController.ts

```diff
@@ -1,15 +1,23 @@
 import { RequestHandler } from 'express'
 import { Gender } from 'welcome'
 import ExpectedArrivalsService from '../services/expectedArrivalsService'
+import ImprisonmentStatusesService from '../services/imprisonmentStatusesService'
```

Comment:
could be importing the type of these services

Reply:
Sorted for next commit